guides/theming.md

```diff
@@ -375,6 +382,38 @@ $my-palette: mat.define-palette(mat.$indigo-palette);
 }
 ```
 
+## Customizing density
+
+Angular Material's density customization is based on the
+[Material Design density guidelines](https://m2.material.io/design/layout/applying-density.html). This system
+defines a scale where zero represents the default density. You can decrement the number for _more density_ and increment
+the number for _less density_.
+
+The density system is based on a *density scale*. The scale starts with the
+default density of `0`. Each whole number step down (`-1`, `-2`, etc.) reduces
+the affected sizes by `4px`, down to the minimum size necessary for a component to render coherently.
```

Comment:
Whole step down, what about whole step up? Or should this be negative values only (<0)?

Reply:
This is only negative values for now.